lib/msal-browser/src/broker/nativeBroker/NativeRequest.ts

```diff
@@ -52,3 +52,30 @@ export type NativeExtensionRequest = {
     extensionId?: string;
     body: NativeExtensionRequestBody;
 };
+
+export type PlatformDOMTokenRequest = {
```

Comment:
Surprising to me that the DOM API requires a different parameter list? I would expect the types here to be the same between DOM and extension given they use the same underlying broker, no?

Reply:
There was a explainer design effort that determined the datatype with Edge folks and for the future work to standardize this across browsers (i.e., have chrome/firefox and safari eventually support this). They want to hide the request complexity into `extraParams` so that they can internally map it to oneAuth as needed. I do not think we have much sway there.